packages/argo-checkout/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -115,7 +117,15 @@ interface AddVariantChange {
   discount?: ExplicitDiscount;
 }
 
-type Changes = AddVariantChange[];
+interface AddShippingLineChange {
+  type: 'add_shipping_line';
+  price: number;
+  title?: string;
+  presentmentTitle?: string;
+  phone?: string;
```

Comment:
`phone` is not in the Docs (we should update the Docs :P)

Reply:
I added the field to the doc, but I'm not sure what the description should be there.